src/app/components/SkipLinkWrapper/index.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { node, string, shape } from 'prop-types';
+import styled from 'styled-components';
+import detokenise from '@bbc/psammead-detokeniser';
+import { visuallyHiddenTextStyle } from '@bbc/psammead-visually-hidden-text';
+import { C_EBON, C_WHITE } from '@bbc/psammead-styles/colours';
+import { getSansBold } from '@bbc/psammead-styles/font-styles';
+import { GEL_BREVIER } from '@bbc/gel-foundations/typography';
+
+const SKIP_LINK_BORDER = '0.125rem';
```

Comment:
As this is the border weight or [width](https://developer.mozilla.org/en-US/docs/Web/CSS/border-width) and not the entire rule, would it make sense to call this `BORDER_WIDTH`? We're inside the SkipLink component and this is the only border, so I also think it makes sense not to namespace/prefix with "skip" and "link".